src/main/java/me/zeroeightsix/kami/command/commands/CoordsCommand.java

```diff
@@ -0,0 +1,85 @@
+package me.zeroeightsix.kami.command.commands;
+
+import me.zeroeightsix.kami.command.Command;
+import me.zeroeightsix.kami.command.syntax.ChunkBuilder;
+import me.zeroeightsix.kami.util.Coord;
+
+import java.util.Objects;
+
+import static me.zeroeightsix.kami.util.CoordUtil.coordsLogToArray;
+import static me.zeroeightsix.kami.util.CoordUtil.writePlayerCoords;
+import static me.zeroeightsix.kami.util.MessageSendHelper.sendChatMessage;
+import static me.zeroeightsix.kami.util.MessageSendHelper.sendRawChatMessage;
+
+/**
+ * @author wnuke
+ * Created by wnuke on 17/04/20
+ */
+
+public class CoordsCommand extends Command {
+    public CoordsCommand() {
+        super("pos", new ChunkBuilder()
+                .append("save|list", true)
+                .append("name|searchterm", false)
+                .append("chunk|showDateTime", false)
+                .build());
+        setDescription("Log the current coordinates.");
+    }
+
+    public void call(String[] args) {
+        if (args[0] != null) {
+            if (args[0].equals("save")) {
+                if (args[1] != null) {
+                    confirm(args[1], writePlayerCoords(args[1], Boolean.parseBoolean(args[2])));
+                } else {
+                    confirm("Unnamed", writePlayerCoords("Unnamed", false));
+                }
+            } else if (args[0].equals("list")) {
+                if (args[1] != null) {
```

Comment:
Again, this will never be null

Reply:
Actually it's null if the person doesn't submit anything as a second argument.

Reply:
Yes, its to make sure there is an argument to avoid errors.

Reply:
What? What? Why? Do you understand how arrays work?

Reply:
Yes, and this works, so it is null when there is not an argument, I have tested this multiple times.